pkg/ebpf/bytecode/tracer-ebpf.go

```diff
@@ -755,7 +755,7 @@ func bindataPkgEbpfCTracerebpfdebugo() (*asset, error) {
 		name:        "pkg/ebpf/c/tracer-ebpf-debug.o",
 		size:        59848,
 		md5checksum: "",
-		mode:        os.FileMode(436),
+		mode:        os.FileMode(420),
```

Comment:
FYI we're going to stop embedding the binaries inside the executable, so these changes should stop appearing